### ISSUE TITLE
Resolve Gravatars via Gravatar?

### ISSUE THREAD

**Issue by RobotLimeLtd** [CDKGlobal/pipeline#2]
I noticed that the Pipeline Plugin uses a linked JIRA application (if one exists) to find the Gravatar, rather than Gravatar directly.  This causes problems if you're not using JIRA, or of your JIRA instance won't resolve gravatar images via the URL http://jira-server/secure/useravatar?ownerId=<jira id>.

Can you resolve Gravatars directly via Gravatar, using the email addess of the Bamboo user?

**Comment by dougmessick** [comment]
Some clarification so I understand your feature request fully:

There are instances where bamboo servers are not able to reach out to external sites for security reasons, so I'm assuming you would expect to have the gravatar links set up in bamboo as in the documentation here:
https://confluence.atlassian.com/display/BAMBOO/Configuring+Gravatar+support. 
Also, just because some users may have gravatars set up, others may only have avatars in Jira. Would it be ok if we only allowed one source of truth instead of trying to fail back to the other and user both?

Another option that came to mind would be for users who don't have avatars set up to have a colored box with their initial in it, similar to how google does when multiple people edit a google doc. Would that be a better fallback for when no avatar is found?

**Comment by RobotLimeLtd** [comment]
Yes, I'm working on the basis that if your Gravatar is visible in Bamboo, it's because your Bamboo server can connect to the default public Gravatar service, or you have set Bamboo up to point to an internal Gravatar server.  In either case, the Pipeline plugin should use this same mechanism for Gravatar display.

I like the idea of User initials in a box to replace Gravatars that cannot be resolved.
